### DISCUSSION TITLE
A newly published site (jekyll) is having problems

### DISCUSSION THREAD

**Discussion by jamesforrest** [community/community#21545] [Other Feature Feedback, Questions, & Ideas]
<p>Hi there guys. So, there’s an issue with viewing a published site. navigating to ‘<a href="http://jamesforrest.github.io" rel="nofollow noopener">jamesforrest.github.io</a>’ shows a 404, even after pushing to the repo:</p>
<pre><code>c:\Users\james\my-portfolio-site&gt;git push -u origin master
Counting objects: 11, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (10/10), done.
Writing objects: 100% (11/11), 3.78 KiB | 484.00 KiB/s, done.
Total 11 (delta 0), reused 0 (delta 0)
To https://github.com/jamesforrest/jamesforret.github.io
 * [new branch]      master -&gt; master
Branch master set up to track remote branch master from origin.
</code></pre>
<p>What can i do to resolve this?</p>

**Comment by winstliu** [comment, marked as answer]
<p>You’ve misspelled your repo name.  It should be <em>jamesforrest</em>.github.io, not <em>jamesforret</em>.github.io.</p>

**Comment by jamesforrest** [comment]
<p>Alrite, deleted previous repo and created a new one. After adding the correct remote, ‘push’ successfully deployed the site.</p>
<pre><code>c:\Users\james\my-portfolio-site&gt;git push -u origin master
Counting objects: 11, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (10/10), done.
Writing objects: 100% (11/11), 3.78 KiB | 352.00 KiB/s, done.
Total 11 (delta 0), reused 0 (delta 0)
To https://github.com/jamesforrest/jamesforrest.github.io.git
 * [new branch] master -&gt; master
Branch master set up to track remote branch master from origin.
</code></pre>
<p>Solved. Thanks!</p>
